### PR TITLE
kubectl ko perf diagnose

### PR DESCRIPTION
**Pull request by changluyi**
- [ ] Make sure you have followed the [Document Convention](../docs/reference/document-convention.md).

### CODE REVIEW

File: docs/ops/kubectl-ko.en.md [hunk 1]
```diff
@@ -658,6 +658,8 @@ I0603 10:35:05.458523   17619 ping.go:83] start to check node connectivity
 
 If the target of diagnose is specified as subnet, the script will create a daemonset on the subnet, and `kube-ovn-pinger` will detect the connectivity and network delay of all pods in this daemonset, and automatically destroy the daemonset after the test.
 
+If the target of diagnose is specified as IPPorts, the script will let each `kube-ovn-pinger` pod detect whether the target protocol, IP, and Port are reachable
```
Comment (oilbeater) [inline docs/ops/kubectl-ko.en.md:661]:
```suggestion
If the target of diagnose is specified as IPPorts, the script will let each `kube-ovn-pinger` pod detect whether the target protocol, IP, and Port are reachable.
```

File: docs/ops/kubectl-ko.md [hunk 2]
```diff
@@ -655,6 +655,8 @@ I0603 10:35:05.458523   17619 ping.go:83] start to check node connectivity
 
 如果 diagnose 的目标指定为 subnet 该脚本会在 subnet 上建立 daemonset，由 `kube-ovn-pinger` 去探测这个 daemonset 的所有 pod 的连通性和网络延时，测试完后自动销毁该 daemonset。
 
+如果 diagnose 的目标指定为 IPPorts 该脚本会让每个 `kube-ovn-pinger` pod 去探测目标协议，IP，Port 是否可达
```
Comment (oilbeater) [inline docs/ops/kubectl-ko.md:658]:
```suggestion
如果 diagnose 的目标指定为 IPPorts 该脚本会让每个 `kube-ovn-pinger` pod 去探测目标协议，IP，Port 是否可达。
```

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: docs/ops/kubectl-ko.en.md (hunk 1 above)
Comment (changluyi) [reply docs/ops/kubectl-ko.en.md]: fixed

File: docs/ops/kubectl-ko.md (hunk 2 above)
Comment (changluyi) [reply docs/ops/kubectl-ko.md]: fixed

**Comment by oilbeater** [comment]
lint failed

**Comment by changluyi** [comment]
> lint failed

fixed

**Comment by changluyi** [comment]
> lint failed

fixed

**Pull request merged**